test/fixtures/fixtures_teams.csv

```diff
@@ -1,6 +1,9 @@
-team_id,franchiseId,teamName,abbreviation,Stadium,link
+team_id,franchiseId,teamName,abbreviation,stadium,link

```

Comment:
This change doesn't much matter since it's in the fixtures.  I double checked and this change didn't happen in the main file.